### PR TITLE
User can rate an article

### PR DESCRIPTION
**Pull request by ESteinman**
## User-story: 
```
As a User,
In order to leave my feedback to the article that I have just read,
I would like to be able to rate it.
```

## Tasks:
- ratings 1-5
- tie rating to specific article
- only Registered users & Subscribers can rate and (not Visitors?)
- save to the DB

### CODE REVIEW

**Comment by tochman** [comment]
You are working on this one, right @ESteinman ?

**Comment by ESteinman** [comment]
I think this one is mergeable. @tochman

File: app/views/articles/show.html.haml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+.notice
+.alert
```
Comment (tochman) [inline app/views/articles/show.html.haml]: I would add these classes to the notice and error divs you have in the application layout

File: features/step_definitions/ratings_steps.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Then("I click {string} on rating") do |value|
+    within('#rating') do
+        click_on value
+    end
+end
+  
+Then("the average rating for {string} should be {string}") do |string, string2|
+
+end
```
Comment (tochman) [inline features/step_definitions/ratings_steps.rb]: Empty step definition

File: features/step_definitions/ratings_steps.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+Then("I should see the notice {string}") do |expected_text|
+    notice = page.evaluate_script("document.querySelector('.notice');").text
+    expect(notice).to eq expected_text
+end
```
Comment (tochman) [inline features/step_definitions/ratings_steps.rb]: Perhaps you should NOT have a separate step for this but rather use `Then I should see..." step?

**Pull request merged**